rules/github-mobile/rule.md

```diff
@@ -0,0 +1,45 @@
+---
+seoDescription: Discover how having GitHub on your phone helps you stay connected to your projects and unblock your team from anywhere.
+type: rule
+title: Do you have GitHub on your phone?
+uri: github-mobile
+authors:
+  - title: Seth Daily
+    url: https://ssw.com.au/people/seth-daily
+related:
+  - pomodoro
+  - github-notifications
+created: 2024-10-05T13:42:55.753Z
+guid: aed50ddb-6105-491e-9963-5cc514766dc9
+---
+
+As a developer you're likely using GitHub all the time - it's one of the main repositories for code. When you're keeping a good work-life balance, it's not ideal for someone to be blocked by a pull request review that would take you 1 minute - just because you're out getting some fresh air.
+
+<!--endintro-->
+
+Having GitHub on your mobile helpw you unblock people quickly, and leave comments all while you’re on the go. Devs are more productive [when they get regular breaks](https://www.ssw.com.au/rules/pomodoro) - so why not review pull requests in the park!
+
+### Why You Should Have GitHub on Your Phone
+
+* **Unblock others** - Nothing is more frustrating for a developer than waiting hours or even days for a small PR approval or a comment on an issue
+
+* **Stay updated** - It’s easier to fall behind on comments and issues if you’re only able to check GitHub from your computer.
+
+* **[Get notifications](https://www.ssw.com.au/rules/github-notifications/)**  GitHub's mobile app lets you control which notifications you receive
+
+### GitHub Contributions Widget
+
+Seeing your contributions can be a great motivator and helps you keep track of your activity. You can add the GitHub contributions widget to your phone’s home screen to view your recent contributions at a glance.
+
+1. **iOS**:  
+   * Tap and hold on your home screen until the icons jiggle.
+   * Tap the “+” icon in the top-left corner and search for “GitHub.”
+   * Choose the GitHub contributions widget and place it on your home screen.
+
+2. **Android**:  
+   * Tap and hold on your home screen and select “Widgets.”
+   * Search for the GitHub contributions widget and add it to your home screen.
+
+::: good
+![Figure: Good example - You can see your contributions from the home screen 🤖](contributions-mobile.png)
```

Comment:
@sethdaily 

- [ ] 1. Cab you make this image medium as per https://www.ssw.com.au/rules/rule

For a mobile screenshot it will look to big vertically in full size